src/utils/utils.ts

```diff
@@ -140,15 +152,13 @@ export const apiCall = async (apiUrl: string) => {
   }
 };
 
-export const getItemDataFromImageID = async (imageID: string) => {
-  console.log(
-    `getItemDataFromImageID: About call getAPIUri and apiCall for image id: ${imageID}`
-  );
-  const apiUri = await getAPIUri("local_image_id", imageID);
-  const data = await apiCall(apiUri.apiUri);
+export const getFeaturedImage = async () => {
+  console.log(`getFeaturedImage: About call getAPIUri`);
+  const apiUri = await getAPIUri("featured", "", { random: "true" });
   return {
-    uuid: apiUri.uuid,
-    title: getTitleFromRepoAPIResponseData(data) || "",
```

Comment:
This method is no longer being used, but I'm not sure if we want to keep it in this file, just in case it's ever needed in the future?

Reply:
Don't think we'll need this so feel free to remove this.